FtcRobotController/src/main/java/com/qualcomm/ftcrobotcontroller/opmodes/SwerveActionDelay.java

```diff
@@ -0,0 +1,60 @@
+package com.qualcomm.ftcrobotcontroller.opmodes;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+/**
+ * Created by Steve on 6/26/2015.
+ */
+public class SwerveActionDelay implements SwerveAction {
```

Comment:
What is the purpose of the "Swerve" prefix? If the intent is to identify Swerve Robotics specific code, I recommend we go with something less ambiguous like "SR_". 

"Swerve" is also a description of a drive mode and possibly the description of other items.